cloud-platform-reports-cronjobs/values.yaml

```diff
@@ -38,3 +38,5 @@ cronjobs:
   infraDeployments:
     name: infrastructure-deployments
     schedule: "59 23 28-31 * *"
+cloudPlatformReports:
```

Comment:
This is deployed in manager on concourse-main namespace. So would still need the value from a kubernetes_secret.

Reply:
can we add a k8s_secret of same name into concourse namespace for that? This value is just a name pointer for the actual deployment lookup:
https://github.com/ministryofjustice/cloud-platform-how-out-of-date-are-we/blob/main/cloud-platform-reports/templates/deployment.yaml#L22-L25